components/shell/ConfigurationFlow.tsx

```diff
@@ -164,7 +164,9 @@ export function ConfigurationFlow() {
         key={config.updatedAt}
         session={session}
         onComplete={(skip) => {
-          client.reconfigure({ skipRecord: skip })
+          client
+            .reconfigure({ skipRecord: skip })
+            .then(() => client.refetchServerConfig())
```

Comment:
I wouldn't expect the server config to be updated from the labeler record configuration flow—is it possible we don't need it?

Reply:
it doesn't update but this is the issue i was seeing https://github.com/bluesky-social/ozone/pull/156#discussion_r1705843465 so after the configuration is complete, we are fetching the server config for real here.